tasks/ms_defender_atp/lib/ms_defender_atp_helper.rb

```diff
@@ -17,14 +17,13 @@ def atp_get_machines(token)
       print_error "Unable to process response!"
     end
 
-  json["value"]
+    json["value"]
   end
   
-  def atp_get_vulns(token,machine_id)
-    print "Getting vulns for machine #{machine_id}"
-    atp_query_api = "https://api.securitycenter.microsoft.com/"
+  def atp_get_vulns(token,atp_query_api)
+    print "Getting vulns"
     headers = {'content-type' => 'application/json', 'accept' => 'application/json', 'Authorization' => "Bearer #{token}", 'accept-encoding' => 'identity'}
-    url =  "#{atp_query_api}/api/machines/#{machine_id}/vulnerabilities"
+    url =  "#{atp_query_api}/api/vulnerabilities/machinesVulnerabilities?skip=10000"
```

Comment:
what does the skip parameter do? Should this be passed to the function so we can paginate?

Reply:
They don't have pagination, this is what they (microsoft) recommended I use.